pkg/filters/options.go

```diff
@@ -120,6 +121,8 @@ func (o *Options) Namespaces(clientset kubernetes.Interface) []string {
 		excludeNamespaces := o.ExcludeNamespaces
 
 		if len(o.IncludeNamespaces) > 0 {
+			slices.Sort(includeNamespaces)
```

Comment:
Any idea why its coming duplicate?

Reply:
Duplication is coming from pflag package and he way it parses argument of type string slice. Duplication is happening here:
https://github.com/spf13/pflag/blob/d5e0c0615acee7028e1e2740a11102313be88de1/string_slice.go#L50
I can't say for sure if it is a bug in the package or there is other logic behind it but looking at the definition of the filed 'changed' 
https://github.com/spf13/pflag/blob/d5e0c0615acee7028e1e2740a11102313be88de1/flag.go#L177
to me it more seems like an error in the logic.